src/transformers/quantizers/quantizer_compressed_tensors.py

```diff
@@ -65,12 +65,19 @@ def _process_model_before_weight_loading(self, model, **kwargs):
         ct_quantization_config = self.compressor.quantization_config
         apply_quantization_config(model, ct_quantization_config, run_compressed=True)
 
-    def _process_model_after_weight_loading(self, model, **kwargs):
+    def _process_model_after_weight_loading(self, model, **kwargs) -> None:
         pass
 
     @property
-    def is_trainable(self):
-        return False
+    def is_trainable(self) -> bool:
+        """Models quantized using compressed tensors can be finetuned"""
+        return True
 
-    def is_serializable(self, safe_serialization=None):
-        return False
+    @property
+    def is_qat_trainable(self) -> bool:
+        """Loaded Models can carry out quantization aware training"""
+        return True
```

Comment:
This shouldn't be true by default no ? It depends how the model was loaded if i'm correct.

Reply:
This depends on whether run_compressed is true or not. This function should be changed to be dependent on that case

Reply:
Sounds good. I don't mind leaving it to True for now but it would be nice to change that in the other PR where you introduce `run_compressed`.